custom_components/tuya_local/devices/pir_spotlight.yaml

```diff
@@ -0,0 +1,184 @@
+name: PIR Spotlight
+products:
+  - id: reldobj3ny6fbgvw
+    name: WiFi Duo LED Light WFD 3050 P
+
+primary_entity:
+  entity: light
+  dps:
+    - id: 20
+      type: boolean
+      name: switch
+    - id: 21
+      type: string
+      name: work_mode
+      mapping:

```

Comment:
It probably doesn't need the mapping, as it doesn't need to match these with HA color_modes, the value is just reported here for reference.

Reply:
Not sure about this one... changing this dps via tinytuya does have some effect (the light dims). Obviously colors don't work but maybe the other values are usable? Although I haven't seen anyting related to the scenes in the app.
Ok, will remove for now and test some more sometime later.